src/stream_management/mod_stream_management.erl

```diff
@@ -104,7 +111,12 @@ stale_h_config_spec() ->
                                                 wrap = {kv, stale_h_repeat_after}},
                   <<"geriatric">> => #option{type = integer,
                                              validate = positive,
-                                             wrap = {kv, stale_h_geriatric}}
+                                             wrap = {kv, stale_h_geriatric}}},
```

Comment:
How about removing the `stale_h` prefixes from these two options?

Reply:
Yeah, I was thinking exactly the same. I left it, because it was just a bit more readable in `mod_stream_management_mnesia` when the options are used, and it's only internal change. But I think I'll remove this wrapping indeed, I'm not a fan of doing this.